File: data/java/FIND_ONE_EXTRA_CHARACTER_STRING_1_TRANSLATED.py
====================
def find_one_extra_character_string_1 ( str_a , str_b ) :
    res , i = 0 , 0
    for i in range ( len ( str_a ) ) :
        res ^= str_a [ i ]
    for i in range ( len ( str_b ) ) :
        res ^= str_b [ i ]
    return ( chr ( res ) , i )

File: data/python/FIND_ONE_EXTRA_CHARACTER_STRING_1_TRANSLATED.java
====================
public static byte fGold ( String strA , String strB ) {
  int res = 0 ;
  for ( int i = 0 ;
  i < strA . length ( ) ;
  i ++ ) {
    res = res ^ ( ( byte ) strA . charAt ( i ) ) ;
  }
  for ( int i = 0 ;
  i < strB . length ( ) ;
  i ++ ) {
    res = res ^ ( ( byte ) strB . charAt ( i ) ) ;
  }
  return ( ( byte ) res ) ;
  ;
}

